compute/computer.go

```diff
@@ -116,6 +119,34 @@ func (c *Computer) ExecuteTransaction(
 	}
 
 	delta := view.Delta()
+	prog := programs.NewEmptyPrograms()
```

Comment:
Already created above (`err = c.vm.Run(ctx, proc, view, programs.NewEmptyPrograms())`), could be re-used